MusicCloud/MusicCloud/Repository/PlaylistRepository.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ErrorResponse.swift
+//  MusicCloud
+//
+//  Created by thanh on 2/26/19.
+//  Copyright © 2019 thanh. All rights reserved.
+//
+
+import Foundation
+import ObjectMapper
+
+protocol PlaylistRepository {
+    func searchPlaylists(keyword: String, limit: Int, completion: @escaping (BaseResult<SearchPlaylistResponse>) -> Void)
+}
+
+class PlaylistRepositoryImpl: PlaylistRepository {
+    private var api: APIService?
```

Comment:
```suggestion
    private var api: APIService!
```
vì sử dụng `required init(api: APIService)` nên sẽ không thể nil nên có thể sử dụng auto unwrap